frontend/src/modules/Environments/views/EnvironmentCreateForm.js

```diff
@@ -370,12 +370,12 @@ const EnvironmentCreateForm = (props) => {
                           color="text.secondary"
                           gutterBottom
                         >
-                          If Using Custom CDK Execution Policy (From Step 1):
+                          If Using Custom CDK Execution Policy (From Step 1. Change the ParameterValue dataall to the Data.all environment resource prefix. Default is dataall.):
                         </Typography>
                         <Typography color="textPrimary" variant="subtitle2">
                           <CopyToClipboard
                             onCopy={() => copyNotification()}
-                            text={`cdk bootstrap --trust ${trustedAccount} -c @aws-cdk/core:newStyleStackSynthesis=true --cloudformation-execution-policies arn:aws:iam::ACCOUNT_ID:policy/DataAllCustomCDKPolicy aws://ACCOUNT_ID/REGION`}
+                            text={`aws cloudformation --region REGION create-stack --stack-name DataAllCustomCDKExecPolicyStack --template-body file://cdkExecPolicy.yaml --parameters ParameterKey=EnvironmentResourcePrefix,ParameterValue=dataall --capabilities CAPABILITY_NAMED_IAM && cdk bootstrap --trust ${trustedAccount} -c @aws-cdk/core:newStyleStackSynthesis=true --cloudformation-execution-policies arn:aws:iam::ACCOUNT_ID:policy/DataAllCustomCDKPolicy aws://ACCOUNT_ID/REGION`}
```

Comment:
When we create the stack, do we need to wait for the stack creation to be complete for the cdk bootstrap command?

Reply:
The command should wait but let me double check this @dlpzx

Reply:
Hi @mourya-33, we are closing the items that are going to be included in v2.1.0. This PR is a little late to be merged in this minor release, but it can be merged as part of a patch release v2.1.1

Reply:
Thanks @dlpzx . I also added the change for renaming CDK toolkit stack name using command line parameter itself while bootstrapping. 
https://github.com/awslabs/aws-dataall/issues/719